### PR TITLE
New docs

### PR DESCRIPTION
**Pull request by RDIL**
## Description  
This updates the docs.  

## Context  
It looks better.  

## Additional Information  
I know this is a bit odd but @73VW do you mind taking a quick glance over at this?  Appreciate it.

### CODE REVIEW

File: area4/__init__.py [hunk 1]
```diff
@@ -1769,6 +1769,7 @@ def div172():
 
 	return divider172
 
+
```
Comment (RDIL) [inline area4/__init__.py]: Ignore changes to this file, just trying to follow PEP-8 for best practices.

File: setup.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="area4",
-    version="1.2.7",
+    version="1.2.8",
```
Comment (RDIL) [inline setup.py]: Bump version.

File: docs/Makefile [hunk 3]
```diff
@@ -8,7 +8,7 @@ SPHINXPROJ 	  = Area4
 SOURCEDIR     = .
 BUILDDIR      = _build
 
-# Put it first so that "make" without argument is like "make help".
+# Put it first so that "make" without argument is the same as "make help".
```
Comment (RDIL) [inline docs/Makefile]: Just a quick grammar thing.

File: docs/customdivs.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    area4.make_div('=-', length=9, start='<', end='=>')    # Returns: '<=-=-=-=>'
+
+    # Resize existing dividers
+    area4.make_div(area4.divider1, 6)    # Returns: '------'
+
+    # Setting to custom div:
+    area4.custom_div = area4.make_div('<>', 24)
+
+    # or directly printing
+    print(area4.make_div('<>', 24))
```
Comment (CoffeeGoddess) [inline docs/customdivs.rst]: Maybe we shouldn't use print for everything, I think it looks messy.

File: docs/examples.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    print(area4.divider9)
+    print(area4.divider10)
+    print(area4.divider11)
+    print(area4.divider12)
+    print(area4.divider13)
+    print(area4.divider14)
+    print(area4.divider15)
+
+
+
```
Comment (CoffeeGoddess) [inline docs/examples.rst]: Remove trailing whitespace.

File: docs/examples.rst [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    print(area4.div8())
+    print(area4.div9())
+    print(area4.div10())
+    print(area4.div11())
+    print(area4.div12())
+    print(area4.div13())
+    print(area4.div14())
+    print(area4.div15())
+
+
```
Comment (CoffeeGoddess) [inline docs/examples.rst]: Same here, too much whitespace.

File: docs/index.rst [hunk 7]
```diff
@@ -10,13 +10,18 @@ Welcome to Area4's documentation!
     :maxdepth: 4
     :caption: Contents:
 
+    installing
+    examples
```
Comment (CoffeeGoddess) [inline docs/index.rst]: I assume this links the other reStructuredText files?

File: docs/index.rst [hunk 8]
```diff
@@ -262,158 +260,22 @@ Divider looks
...
-To install via requirements.txt  
--------------------------------
-
-To use area4 as a dependency for your project, you can add the following line:  
-
-.. code-block:: console
-
-    area4 
-
-You must have prior knowledge with using a requirements.txt file to take this path.
```
Comment (CoffeeGoddess) [inline docs/index.rst]: Maybe we should add more info on using the requirements file, not all people know a lot about pip.

File: docs/index.rst [hunk 9]
```diff
@@ -262,158 +260,22 @@ Divider looks
...
-    print(area4.divider9)
-    print(area4.divider10)
-    print(area4.divider11)
-    print(area4.divider12)
-    print(area4.divider13)
-    print(area4.divider14)
-    print(area4.divider15)
-    
-   
-
```
Comment (CoffeeGoddess) [inline docs/index.rst]: Remove trailing whitespace.

File: docs/index.rst [hunk 10]
```diff
@@ -262,158 +260,22 @@ Divider looks
...
 
-Variable:
-dividerNumber = str("dividertexthere")
+.. code-block:: python
+
+    # Variable:
+    dividerNumber = str("dividertexthere")
 
-def divNumber():
-{indent}return dividerNumber
```
Comment (CoffeeGoddess) [inline docs/index.rst]: That looks ugly.  Thank goodness it is gone.

**Comment by 73VW** [comment]
Hello @RDIL,
I'm sorry but I haven't had time yesterday to look at it. I have just seen that you merged it. Do you still need my help?
Cheers

**Comment by RDIL** [comment]
Oh, no we’re good now, thanks anyway!

**Comment by 73VW** [comment]
Happy to hear that !
Have a nice day !

**Pull request merged**